shared-bindings/microcontroller/Processor.c

```diff
@@ -61,6 +72,27 @@
 //|     frequency: int
 //|     """The CPU operating frequency in Hertz. (read-only)"""
 //|
+
+STATIC mp_obj_t mcu_processor_set_sys_clock(mp_obj_t self_in, mp_obj_t freq) {
+    mcu_processor_obj_t *self = MP_OBJ_TO_PTR(self_in);
+    #if defined(HAS_SETTABLE_CLOCK)
+    uint32_t value_of_freq = MP_OBJ_SMALL_INT_VALUE(freq);
+    common_hal_mcu_processor_set_sys_clock(self, value_of_freq);
+    #else
+    mp_raise_msg(&mp_type_NotImplementedError,translate("Settable Clock Not Implemented for Your Board"));
+    #endif
+    return mp_const_none;
+}
+
+MP_DEFINE_CONST_FUN_OBJ_2(mcu_processor_set_sys_clock_obj, mcu_processor_set_sys_clock);
+
+const mp_obj_property_t mcu_processor_freq_obj = {
+    .base.type = &mp_type_property,
+    .proxy = {(mp_obj_t)&mcu_processor_set_sys_clock_obj,
+              MP_ROM_NONE,
+              MP_ROM_NONE},
+};
```

Comment:
Restore this to the old way, and add a setter.
```suggestion
const mp_obj_property_t mcu_processor_freq_obj = {
    .base.type = &mp_type_property,
    .proxy = {(mp_obj_t)&mcu_processor_get_frequency_obj,
              (mp_obj_t)&mcu_processor_set_frequency_obj,
              MP_ROM_NONE},
};
```

Reply:
Cool didn't even realize that I did that.  

And before I forget thanks - learning something with this exercise with you walking me through things.

Reply:
Thank you for persevering!